customer/kubernetes/Deployment.remote.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: customer
+    version: v1
+  name: customer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: customer
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: customer
+        version: v1
+    spec:
+      containers:
+      - env:
+        - name: JAVA_OPTIONS
+          value: -Xms128m -Xmx256m -Djava.net.preferIPv4Stack=true -Djava.security.egd=file:///dev/./urandom
+        image: quay.io/rhdevelopers/istio-tutorial-customer:latest
```

Comment:
Since it is public image I think that not using the `latest` label might be better in the future.

Reply:
good point! this is just to go with whats in tutorial

Reply:
+1

Reply:
Maybe customer:v1

Reply:
ok will update the PR